sdk/core/core-http-compat/review/core-http-compat.api.md

```diff
@@ -21,11 +23,14 @@ export interface CompatResponse extends Omit<FullOperationResponse, "request" |
     request: WebResourceLike;
 }
 
+// @public
+export function convertHttpClient(requestPolicyClient: RequestPolicy): HttpClient;
+
 // @public
 export function createRequestPolicyFactoryPolicy(factories: RequestPolicyFactory[]): PipelinePolicy;
 
 // @public (undocumented)
-export const disbaleKeepAlivePolicyName = "DisableKeepAlivePolicy";
+export const disableKeepAlivePolicyName = "DisableKeepAlivePolicy";
```

Comment:
is this breaking?

Reply:
Yep! As far as I can tell nobody ever imported it, but we can export the typo name as `@deprecated` too

Reply:
I am tempted to major version the compat lib due to the big changes though

Reply:
bumped the version to `2.0.0`